### PR TITLE
Adding a section about new releases blog 

### LINKED ISSUE #1880: Update the "releases" section of the Blog
[Blog -> Releases](https://knative.dev/blog/releases/)

Determine how to use this section going forward. It is outdated and excludes mention of any of the recent releases.

Will contributors continue to add posts here for each release?

### PR DESCRIPTION
**Pull request by macruzbar**
Fixes #1880  or description of the problem the PR solves

## Proposed Changes 

- Added a section to the blogging guidelines that addresses blogs about releases and new features
-
-

### CODE REVIEW

File: blog/README.md [hunk 1]
```diff
@@ -44,6 +44,10 @@ To submit a blog post, follow the steps below.
 - Content that doesn't interact with Knative APIs or interfaces
 - Vendor pitches
 
+## Communicating new project releases
+**Scheduled releases (every 6 weeks).** Knative project has a release every 6 weeks, and we need your help communicating new changes to our community! If you would like to contribute a blogpost to the Knative blog, please consider writing about the latest changes to the project. Ideally, we should have a single blogpost for every release *number*, for example, 0.17; 0.18; 0.19. The title convention should be: *Version [version number] release - [date]*. Release blog contributors should write a summary of the changes, which can be organized by working groups if this makes sense, and select up to 3 highlights of the current release.
```
Comment (abrennan89) [inline blog/README.md:48]:
```suggestion
**Scheduled releases:** The Knative project has a release every 6 weeks, and we need your help communicating new changes to our community! If you would like to contribute a blog post to the Knative blog, please consider writing about the latest changes to the project. Ideally, there should be a single blog post for every release version, for example, 0.17; 0.18; 0.19. The title convention should be: *Version [version number] release - [date]*. Release blog contributors should write a summary of the changes and select up to 3 highlights of the current release to write about.
```

Comment (abrennan89) [reply blog/README.md:48]: This part didn't really make sense to me so I suggested removing the bit mentioning working groups and some other minor changes.

**Review by abrennan89 — COMMENTED** [review]
Added some suggestions, otherwise lgtm

**Comment by RichieEscarez** [comment]
Currently, this page is hidden and viewable from only here in this folder from the GitHub repo. It does not get published to knative.dev but is that desired? We previously put all contributor details/instructions in the knative/community repo (which gets published under the Community section in knative.dev [here](https://knative.dev/community/contributing/docs/)). 

If we do want this info publicly visible, we can copy this info to a file over there under the /docs folder and then add a link in this README to crosslink and people to the info? Related: https://github.com/knative/community/pull/205#issuecomment-670674088

**Comment by grantr** [comment]
Verified that both authors (@macruzbar and @abrennan89) have previously signed the CLA.

**Comment by macruzbar** [comment]
> Currently, this page is hidden and viewable from only here in this folder from the GitHub repo. It does not get published to knative.dev but is that desired? We previously put all contributor details/instructions in the knative/community repo (which gets published under the Community section in knative.dev [here](https://knative.dev/community/contributing/docs/)).
> 
> If we do want this info publicly visible, we can copy this info to a file over there under the /docs folder and then add a link in this README to crosslink and people to the info? Related: [knative/community#205 (comment)](https://github.com/knative/community/pull/205#issuecomment-670674088)

Hi @RichieEscarez thank you for your comment. I read your comment in the PR you shared as well, and I think you are raising a good point. The logic I followed is from my training in open leadership at Mozilla, where they encourage to estate in a README.md file the purpose of the folder / repo that that file is in, and how to contribute to that directory. This is what inspired me to add these contribution guidelines here. 

I do think, however, that we need to give all these avenues for contribution more visibility in CONTRIBUTING.md file, and possibly, also on the README.md file on the Community repo. I think adding a link with a brief text about the different ways of contributing could be a good idea, as it gives appropriate framing to the new links added. Let me know if you would be able to take care of this; otherwise, I can take care of it towards the end of the week.

File: blog/README.md [hunk 2]
```diff
@@ -44,6 +44,10 @@ To submit a blog post, follow the steps below.
 - Content that doesn't interact with Knative APIs or interfaces
 - Vendor pitches
 
+## Communicating new project releases
+**Scheduled releases:** The Knative project has a release every 6 weeks, and we need your help communicating new changes to our community! If you would like to contribute a blog post to the Knative blog, please consider writing about the latest changes to the project. Ideally, there should be a single blog post for every release version, for example, 0.17; 0.18; 0.19. The title convention should be: *Version [version number] release - [date]*. Release blog contributors should write a summary of the changes and select up to 3 highlights of the current release to write about.
+**Big changes to the project.** Big changes to the project require a deep dive blog that describe the new feature in detail and give examples of the new functionality.
```
Comment (mpetason) [inline blog/README.md:49]:
```suggestion
**Big changes to the project.** Big changes to the project require a deep dive blog that describes the new feature in detail and give examples of the new functionality.  
```
I think that might read better, although not by much, looks good!

**Comment by RichieEscarez** [comment]
> > Currently, this page is hidden and viewable from only here in this folder from the GitHub repo. It does not get published to knative.dev but is that desired? We previously put all contributor details/instructions in the knative/community repo (which gets published under the Community section in knative.dev [here](https://knative.dev/community/contributing/docs/)).
> > If we do want this info publicly visible, we can copy this info to a file over there under the /docs folder and then add a link in this README to crosslink and people to the info? Related: [knative/community#205 (comment)](https://github.com/knative/community/pull/205#issuecomment-670674088)
> 
> Hi @RichieEscarez thank you for your comment. I read your comment in the PR you shared as well, and I think you are raising a good point. The logic I followed is from my training in open leadership at Mozilla, where they encourage to estate in a README.md file the purpose of the folder / repo that that file is in, and how to contribute to that directory. This is what inspired me to add these contribution guidelines here.
> 
> I do think, however, that we need to give all these avenues for contribution more visibility in CONTRIBUTING.md file, and possibly, also on the README.md file on the Community repo. I think adding a link with a brief text about the different ways of contributing could be a good idea, as it gives appropriate framing to the new links added. Let me know if you would be able to take care of this; otherwise, I can take care of it towards the end of the week.

Sounds good. I too am busy and wont be able to address this until possibly later than that so I'll leave this in your hands. Let's work out a plan (next Docs WG meeting?) for all the various files and figure out how to get "what" into the site and "where" with cross linking, etc (and then write it all down somewhere too - my current priority is documenting the site publishing mechanisms). Thanks @macruzbar !

**Comment by macruzbar** [comment]
this is ready for approval @RichieEscarez @abrennan89 @mpetason

**Review by abrennan89 — APPROVED** [review]
/approve
/lgtm

**Pull request merged**